src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionMod3Case5.java

```diff
@@ -0,0 +1,52 @@
+package org.umssdiplo.automationv01.stepdefinitionproject;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import org.umssdiplo.automationv01.core.managepage.smartHouse.Header;
+import org.umssdiplo.automationv01.core.managepage.smartHouse.SHEmployee;
+import org.umssdiplo.automationv01.core.managepage.smartHouse.SHLogin;
+import org.umssdiplo.automationv01.core.managepage.smartHouse.SHOrganization;
+import org.umssdiplo.automationv01.core.utils.LoadPage;
+
+public class StepsDefinitionMod3Case5 {
+    private SHLogin login;
+    private SHOrganization organization;
+    private Header menu;
```

Comment:
if you are going to use prefix in your Page Objects, you will need to do in the Header too.